### ISSUE TITLE
react-native-themis - Sealed encryption crashes for empty string

### ISSUE THREAD

**Issue by tom-at-pixel** [cossacklabs/themis#926]
**Describe the bug**
When `secureCellSealWithSymmetricKeyEncrypt64()` is called on an empty string, a native exception is thrown. If this is by design, it should be documented. Themis should consider handling this condition internally, since empty fields are a common occurrence for field-level encryption scenarios (e.g. imagine a user creating a to-do with an empty description field, and all user content-generated fields are encrypted).

**To Reproduce**
1. In a React Native project with `react-native-themis` installed, run the following code:

```typescript
const key = "ANY_BYTE_ARRAY";
const emptyText = "";

// This code will throw a native exception
await secureCellSealWithSymmetricKeyEncrypt64(key, emptyText);
```

2. Observe the exception:

![image](https://user-images.githubusercontent.com/109755647/180276227-1f04f567-c670-48d1-aee8-2e7d35b7021c.png)

**Expected behavior**
The encryption should no-op in Themis for empty strings.

**Environment (please complete the following information):**
- OS: iOS 15.2
- Hardware: iOS Simulator
- Themis version: react-native-themis 0.14.4
- Installation way:
   - [x] via package manager
   - [ ] built from source

**Comment by vixentael** [comment]
Thank you, @tom-at-pixel! We'll take a look.

**Comment by vixentael** [comment]
i guess it's fixed in the latest release 0.14.7
https://github.com/cossacklabs/themis/releases/tag/0.14.7

@tom-at-pixel please take a look

**Comment by tom-at-pixel** [comment]
Thanks @vixentael, that's great!
